### DISCUSSION TITLE
用第三方工具突然显示被封禁

### DISCUSSION THREAD

**Discussion by Pexiong** [BurnHair/docs#111] [技术支持]
![360截图20240620124740](https://github.com/BurnHair/docs/assets/167086571/4d902bfb-6ad0-4f58-8c67-853feefd1c02)

**Comment by BennyThink** [comment]
你问了很多NSFW的问题，因此被封禁。

**Comment by jiangxiaobo1994** [comment]
我也是，我的应该是MP的辅助识别功能，导致的。
